README.md

```diff
@@ -12,15 +12,15 @@
 - **Technologies used**: NPM, Webpack, & Jest Library.
 - **Tools**: Git, Gitflow, GitHub, Netlify, VS Code, & Chrome Browser.
 
-## Live Demo (Not Yet) 🚀
+## Live Demo (Deployed On Netlify) 🚀
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/68ea255b-ae38-4005-968b-6fbd0a8d676b/deploy-status)](#)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/68ea255b-ae38-4005-968b-6fbd0a8d676b/deploy-status)](https://635471502ef0e179bf2f6409--boisterous-dango-e226ad.netlify.app/)
 
-[Live Demo Link](#)
+[Live Demo Link](https://635471502ef0e179bf2f6409--boisterous-dango-e226ad.netlify.app/)
 
 ## Video Link (Recorded by Loom) 📹
 
-[Video Link](#)
+[Video Link](https://loom.com/share/86664c4cbbea4828a9eb5ca11f5eccc0)
```

Comment:
-  I added this comment here cause I don't have where to comment

- [ ] It would be better if you write unit tests for pure function of your redux code and also integration tests for your application to cover it and match the requirements 
- Screenshot of the requirements 
<img width="893" alt="Screen Shot 2022-10-23 at 00 07 55" src="https://user-images.githubusercontent.com/61790587/197365649-b31334ca-c018-4a4b-881b-2ca3c0680d5e.png">

Reply:
Hi @youmari , Thanks for your feedback. Please can you send me the second opinion form so that I can submit the project.

Reply:
Hi @youmari , I have already updated the `tests` could you please give it another look.

Reply:
@ZikaZaki I'm really sorry about I've already reached my hours limit but just send it for review you find other that will review your project